rustls/src/client/tls13.rs

```diff
@@ -1527,6 +1527,7 @@ impl ExpectTraffic {
             protocol: cx.common.protocol,
             quic: &cx.common.quic,
         };
+        cx.common.tls13_tickets_received += 1;
```

Comment:
It might be unwarranted paranoia, but I'm idly wondering whether this should be a saturating or wrapping add to avoid a panic on overflow. I don't think there is a session ticket limit enforced earlier in this codepath, though I'm also not confident it's feasible to transmit ~four billion tickets on one conn :laughing: 

What do other folks think?

Reply:
Hah... I guess it can't hurt to do a `saturating_add()` here?